controllers/feeds/getFeedsV2.js

```diff
@@ -94,6 +94,47 @@ const isValidActivity = async (item, conditions) => {
   return (item.final_score || 0) >= threshold;
 };
 
+const getActorFromLatestReaction = (latestReaction) => {
+  let postActors = new Set();
+  latestReaction?.upvotes?.forEach((upvote) => postActors.add(upvote.user_id));
+  latestReaction?.downvotes?.forEach((downvote) => postActors.add(downvote.user_id));
+  latestReaction?.comments?.forEach((comments) => postActors.add(comments.user_id));
+  latestReaction?.comment?.forEach((comment) => postActors.add(comment.user_id));
+  return Array.from(postActors);
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getActorFromLatestReaction` is using optional chaining (`?.`) to access properties of `latestReaction`. This is good for avoiding errors when the properties do not exist. However, it's important to note that if `latestReaction` itself is `undefined`, this function will return an empty array. If this behavior is not intended, you might want to add a check at the beginning of the function to ensure `latestReaction` is defined.



<!-- This is an auto-generated comment by OSS CodeRabbit -->